assets/scripts/build_gallery.py

```diff
@@ -321,9 +329,38 @@ def get_expectation_instances(expectations_info):
             )
             expectation_tracebacks.write(traceback.format_exc())
             continue
+        except pydantic.ValidationError as exc:
+            logger.error(exc)
+            continue
```

Comment:
I don't think this continue is doing anything. And should the other `except`s reraise?

Reply:
we don't want to raise an exception here, since it'll kill the script. if an expectation breaks when we try to build its docs, we just skip it.

you're right about the `continue`, will remove.